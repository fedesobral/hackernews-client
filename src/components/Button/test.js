import React from 'react';
import ReactDOM from 'react-dom';
// import { shallow, render, mount } from 'enzyme';
// import { renderToJson } from 'enzyme-to-json';
import Button from '../Button';

describe('Button', () => {

	it('renders', () => {
	  const div = document.createElement('div');
	  ReactDOM.render(<Button>Button</Button>, div);
	});
});