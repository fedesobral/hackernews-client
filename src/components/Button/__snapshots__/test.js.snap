exports[`Button Snapshot className button 1`] = `
<button
  className="awesome-btn"
  onClick={undefined}
  type="button">
  Click you
</button>
`;

exports[`Button Snapshot click me button 1`] = `
<button
  className=""
  onClick={undefined}
  type="button">
  Click me
</button>
`;
